src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1541,12 +1529,17 @@ public override BoundNode VisitCall(BoundCall node)
                 // PROTOTYPE(NullableReferenceTypes): Update method based on inferred receiver type.
             }
 
-            ImmutableArray<Result> results = VisitArgumentsEvaluate(node.Arguments, node.ArgumentRefKindsOpt, method, node.ArgsToParamsOpt, node.Expanded);
-            if (method.IsGenericMethod && HasImplicitTypeArguments(node))
+            if (!node.HasErrors)
```

Comment:
>!node.HasErrors [](start = 16, length = 15)

Instead, it probably makes sense to reuse logic used in CSharpOperationFactory.CreateBoundCallOperation to detect if we have reliable mapping from arguments to parameters. We might be able to pull it into a helper method on BoundCall.

Reply:
Added a `// PROTOTYPE` comment.

---
In reply to: [181197485](https://github.com/dotnet/roslyn/pull/26107#discussion_r181197485) [](ancestors = 181197485)